x/application/keeper/msg_server_undelegate_from_gateway_test.go

```diff
@@ -92,6 +95,9 @@ func TestMsgServer_UndelegateFromGateway_SuccessfullyUndelegate(t *testing.T) {
 	_, err = srv.UndelegateFromGateway(ctx, undelegateMsg)
 	require.NoError(t, err)
 
+	sdkCtx.WithBlockHeight(4)
```

Comment:
Thanks for pointing this out. Setting the height and pruning undelegations are no longer needed in this test, since it's only asserting on `EventRedelegation` event emission and `DelegateeGatewayAddresses` being updated (which is no longer delayed)